PowerEditor/src/ScitillaComponent/ScintillaEditView.h

```diff
@@ -95,6 +95,9 @@ const int CP_GREEK = 1253;
 #define LIST_4 16
 #define LIST_5 32
 #define LIST_6 64
+#define LIST_7 128
+#define LIST_8 256
+#define LIST_9 512

```

Comment:
`You can set up to 9 lists of keywords for use by the current lexer. keyWordSet can be 0 to 8:` 
https://www.scintilla.org/ScintillaDoc.html#SCI_SETKEYWORDS

Reply:
So LIST_9 is useless.

Reply:
@donho LIST_9 is removed.